core/sqf/monitor/linux/process.cxx

```diff
@@ -1317,7 +1318,11 @@ bool CProcess::Create (CProcess *parent, int & result)
     env = getenv ("TERM");
     STRCPY (term, (env?env:"ansi"));
     env = getenv ("TZ");
-    STRCPY (tz, (env?env:""));
+    tz_exists = (env != NULL);
+    if (tz_exists)
+    {
+      STRCPY (tz, env); // see note regarding TZ below
+    }
     env = getenv ("USER");
     STRCPY (user, (env?env:""));
     env = getenv ("HOME");
```

Comment:
Change looks good. @selvaganesang has a good point on making the calls to getenv() on each process create. Will have to analyze the impact of caching these environment variables.